docs/sources/features/datasources/influxdb.md

```diff
@@ -141,7 +141,7 @@ To add a filter click the plus icon to the right of the `Measurements/Fields` bu
 ## Templating
 
 Instead of hard-coding things like server, application and sensor name in you metric queries you can use variables in their place.
```

Comment:
```suggestion
Instead of hard-coding things like server, application and sensor name in your metric queries you can use variables in their place.
```